google/auth/impersonated_credentials.py

```diff
@@ -232,7 +232,7 @@ def __init__(
         self._target_principal = target_principal
         self._target_scopes = target_scopes
         self._delegates = delegates
-        self._lifetime = lifetime
+        self._lifetime = lifetime or _DEFAULT_TOKEN_LIFETIME_SECS
```

Comment:
it's already the default value

Reply:
There are a few places where the function might be called with None instead of no value, and I think this works as a decent safety valve